src/modules/project/components/ProjectLists/ProjectLists.tsx

```diff
@@ -1,165 +1,207 @@
-import React, { useEffect } from "react";
-import { useState } from "react";
-import { axiosInstance,  PROJECT_URL } from "../../../../services/EndPoints";
+import React, { useState, useEffect } from "react";
+import { axiosInstance, PROJECT_URL } from "../../../../services/EndPoints";
 import { Button, Dropdown, Modal } from "react-bootstrap";
 import { toast } from "react-toastify";
+import "../ProjectLists/TasksAndProjectsLists.css";
 
-
-interface projectData {
-  title:string;
+interface ProjectData {
+  title: string;
   description: string;
-  task:object[];
-  id:number;
-} 
+  task: object[];
+  id: number;
+}
 
 export default function ProjectLists() {
-  const [projectsList, setProjectsList] = useState([]);
-  const [titleValue , setTitleValue] = useState("");
+  const [projectsList, setProjectsList] = useState<ProjectData[]>([]);
```

Comment:
It is better to use search params for filteration and pagination as it will enable storing the search criteria while reloading